frontend/src/components/TableSkeleton/styles.scss

```diff
@@ -0,0 +1,70 @@
+@use '@bcgov-nr/nr-theme/design-tokens/colors.scss' as colors;
+@use '@bcgov-nr/nr-theme/design-tokens/variables.scss' as vars;
+.activity-table {
+  margin-bottom: 2.5rem;
+
+  tr > th:first-child,
+  tr > td:first-child {
+    padding-left: 2.5rem;
+  }
+
+  tr > th:last-child,
+  tr > td:last-child {
+    padding-right: 2.5rem;
+  }
+
+  tr > th:last-child div,
+  tr > td:last-child {
+    text-align: center;
+  }
+
+  .activities-table-cell svg {
+    position: relative;
+    margin-right: 0.5rem;
+    top: 0.1875rem;
+  }
+}
+
+.#{vars.$bcgov-prefix}--data-table thead tr th#blank {
+  min-width:50px;
+}
+
+.#{vars.$bcgov-prefix}--data-table thead tr th  {
+  background-color: #F3F3F5;
+  border-top: 1px solid;
+  border-color: var(--#{vars.$bcgov-prefix}-border-subtle-01);
+  background-color: var(--#{vars.$bcgov-prefix}-layer-accent-01);
+}
+
+.#{vars.$bcgov-prefix}--data-table thead tr th {
+  min-width:158px;
```

Comment:
question (non-blocking): are we sticking to `rem`?

Reply:
Boo!  px = 666!  :D